webapp/src/css/inbox.less

```diff
@@ -1589,28 +1778,64 @@ mm-sidebar-menu .mat-sidenav-container {
         margin: 0 12px 0 0;
       }
 
+      .open-sort-label {
+        display: none;
+      }
+
       .btn.open-filter {
         margin: 0;
+        padding: 0 5px;
 
         .filter-counter {
           position: absolute;
-          top: 3px;
-          right: -2px;
+          top: -6px;
+          right: 1px;
           border: 1px solid @chip-border-color;
         }
       }
+    }
+  }
 
-      &.open-search {
-        background-color: rgba(255, 255, 255, 0.8);
+  .app-root.search-bar-active mm-search-bar {
+    width: 100%;
+    padding-left: 5px;
+    max-width: initial;
```

Comment:
L1174 is already setting `max-width: initial;`, are you sure we still need it here?